src/main/java/mate/service/CarService.java

```diff
@@ -10,4 +10,6 @@ public interface CarService extends GenericService<Car> {
     void removeDriverFromCar(Driver driver, Car car);
 
     List<Car> getAllByDriver(Long driverId);
+
+    Car registerNewCar(String model, Long manufacturerId);
```

Comment:
This logic is confusing. You already have method 'create' that saves car to the database. And you have method 'register' that creates new car but does not save it into database. It's not obvious from method's name, what does it do. 
In this homework it's better not to create 'register' method. Create new Car in Controller.
Also remove 'register' method from other Services

Reply:
I thought it is a good idea, if we will want to add some check style for inputed data.